light_node/src/configuration.rs

```diff
@@ -423,6 +423,10 @@ pub fn tezos_app() -> App<'static, 'static> {
             .long("disable-mempool")
             .global(true)
             .help("Enable or disable mempool"))
+        .arg(Arg::with_name("disable-blacklist")
```

Comment:
I vote for `disable-peer-blacklist` and "Disable peer blacklisting" as the description.